dd-java-agent/instrumentation/spark/src/testFixtures/java/datadog/trace/instrumentation/spark/TestSparkComputation.java

```diff
@@ -52,4 +65,24 @@ public static Dataset<String> applyIdentityMapFunction(Dataset<String> ds) {
   public static String getSparkVersion() {
     return org.apache.spark.package$.MODULE$.SPARK_VERSION();
   }
+
+  public static SparkSession createSparkSession(String appName) {
+    return SparkSession.builder()
+        .appName(appName)
+        .config("spark.master", "local[2]")
+        .config("spark.sql.shuffle.partitions", "2")
+        .getOrCreate();
+  }
+
+  public static SparkSession createDatabricksWorkflowsSparkSession(String appName) {
```

Comment:
nit: 
```suggestion
  public static SparkSession createDatabricksSparkSession(String appName) {
```

since this spark session is tied to Databricks, but not necessarily to a workflow